### ISSUE TITLE
Update to latest commons-configuration2

### ISSUE THREAD

**Issue by Virtlink** [Virtlink/commons-configuration2-jackson#1]
Currently this project uses `org.apache.commons:commons-configuration2:2.0-beta2`, which is currently not found in any Maven repository. See also [this issue](https://issues.apache.org/jira/browse/CONFIGURATION-606).

The project must be updated to the latest commons-configuration2 release that is found in Maven Central. Until then, we host Apache's Commons Configuration 2 in our own repository at `http://artifacts.metaborg.org/content/repositories/releases/`.

**Comment by heruan** [comment]
Thank you @Virtlink! I'm trying to install your project, I've added it to my `pom.xml` but I get a:

```
Missing artifact org.apache.commons:commons-configuration2:jar:2.0-beta2
```

What should I put in `pom.xml` to get the right dependency?

**Comment by Virtlink** [comment]
This project is an extension of Apache's Commons Configuration version 2 (`org.apache.commons`:`commons-configuration2`) but unfortunately as that's still a beta version, it isn't hosted on Maven Central. Until it is, you can get it from our own repository, by adding this to your `pom.xml`:

```
<repositories>
  <repository>
    <id>metaborg-release-repo</id>
    <url>http://artifacts.metaborg.org/content/repositories/releases/</url>
  </repository>
</repositories>
```

(As you know, beta software shouldn't be used in a production environment, which is why the dependency isn't on Maven Central yet.)

**Comment by Virtlink** [comment]
Fixed now in commit fee95c3: now we use the Apache Commons Configuration2 dependency from Maven Central, since release 0.5.0.
